### ISSUE TITLE
Property name conversion issue

### ISSUE THREAD

**Issue by lEugineer** [mikolajmitura/java-properties-to-json#54]
Hi

Is it possible to configure your tool so that it converts properties:
```
search.index.mapping[server.model.db.Task].indexFields[2].type=text
search.index.mapping[server.model.db.Task].indexFields[2].name=name
```
with a key like **.mapping[server.model.db.Task].** to:
```
"search": {
  "index": {
    "mapping[server.model.db.Task]": {
      "indexFields": [
        ...
        {
          "type": "text", 
          "name": "name"
        }
        ...
      ]
    }
  }
}
```

As the default behaviour converts it to:
```
"search":{
  "index":{
     "mapping[server":{
        "model":{
           "db":{
              "Task]":{
                "indexFields": [
                  ...
                  {
                    "name": "name",
                    "type": "text"
                  }
                  ...
                ]
              }
           }
        }
     }
  }
}
```
I'm using 
```
new PropertiesToJsonConverter().convertToJson(Map<String, String>)
```

**Comment by mikolajmitura** [comment]
Hi, thanks for you issue, 
the value in json **"mapping[server.model.db.Task]"** is your expected?
I'm curious how it will be used in frontend? 

At this moment it is not configurable... I will need prepare fix for that.

**Comment by mikolajmitura** [comment]
I released 5.1.3 version to public maven repository. 
So you can verify that is ok with your requirements.

**Comment by lEugineer** [comment]
Yes, you understood everything correctly.
It works as expected now.
Thank you for new version.
